### ISSUE TITLE
[YAML to PHP] Comments bug

### ISSUE THREAD

**Issue by TavoNiievez** [migrify/migrify#126]
In a `config/packages/framework.yaml`:
```
framework:
  default_locale: '%locale%'
  #http_method_override: true
  #trusted_hosts: ~
```
when executing:
```
vendor/bin/config-transformer switch-format config/packages/framework.yaml --input-format yaml --output-format php --target-symfony-version 5.1
```

with comments starting with `#` I see:
```                                                                                                                           
Malformed inline YAML string: "'http_method_override: true" at line 5 (near "__COMMENT__1: 'http_method_override: true").  
```
removing the comments solves the problem, but the expected behavior is that these lines are ignored in the transformation

**Comment by TomasVotruba** [comment]
Thanks for reporting.

Most comments are ignored, or tried to preserve.
Could you narrow it down to minimal lines causing this problem?

**Comment by TavoNiievez** [comment]
In fact, in my case having two lines of comments in a row already generates the described error. (I updated the issue with the min. config).
but... I also tried to leave a space... And same error:
```
framework:
  default_locale: '%locale%'
  #trusted_hosts: ~

  property_info:
    enabled: true
```
Results in:
```
  Malformed inline YAML string: "'trusted_hosts: ~" at line 3 (near "__COMMENT__1: 'trusted_hosts: ~").  
```

**Comment by TavoNiievez** [comment]
and... maybe this is not a bug itself, but to convert my `config/packages/framework.yaml` to php, I had to manually add this line: `$container->import('../config/{packages}/*.php');` to my `src/Kernel.php`, since until today, the framework-bundle recipe only includes import of .yaml files inside packages/ folder [ symfony/recipes reference](https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/5.1/src/Kernel.php)

**Comment by TomasVotruba** [comment]
I've just noticed the indent is just 2 spaces instead of 4 we use in tests.

Does is have any influence if you use 4 spaces for indent?

**Comment by TavoNiievez** [comment]
I tried it, and I get the same error as a result with 4 space indentation.
Both for the two comments in a row, and for the space after comment.

**Comment by TavoNiievez** [comment]
@TomasVotruba there's a new chapter in this story 😆, Given:

```
fos_ck_editor:
    configs:
        config:
            uiColor: "#F5F5F5"
```
**`#`** is interpreted as a comment even if it's in quotes:
```
 Malformed inline YAML string: "'F5F5F5"" at line 7 (near "__COMMENT__1: 'F5F5F5""). 
```

**Comment by TomasVotruba** [comment]
In the end, it would be better not to have comments and have working tool :smile:  See #129 


Fixed and released in 0.3.20
